QuizApp/QuizApp/UI/Screens/SolvingQuiz/QuestionsView.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+
+class QuestionsView: UIView {
+
+    weak var delegate: QuestionsViewDelegate!
+
+    private var questions: [Question] = []
+
+    private var collectionViewLayout: UICollectionViewFlowLayout!
+    private var collectionView: UICollectionView!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(questions: [Question]) {
+        self.questions = questions
+        collectionView.reloadData()
+    }
+
+    func scrollToQuestion(at index: Int?) {
```

Comment:
Should you be able to scroll to a question if you do not know where to scroll? In other words, should the index be optional?

Reply:
Solved on next branch